### ISSUE TITLE
Sliders should be keyboard accessible

### ISSUE THREAD

**Issue by shawnbot** [18F/college-choice#945]
In #933 I detailed some of the issues with getting the slider interactivity working in IE11. I eventually decided to nix keyboard accessibility in favor of getting it working with mouse and touch working generally, so this issue is about restoring keyboard accessibility.

The issue appears to be that IE11 dispatches `focus` and `blur` events when you engage an element with the `tabindex` attribute set, but it could be more complicated than that.

Specifically, when you focus either of the handles via <kbd>tab</kbd>, the left and right keys (and maybe up and down) should move that handle left or right, modify the respective property (`min` or `max`, respectively) and dispatch a `change` event.

**Comment by shawnbot** [comment]
This will probably need an hour or two of looking at the JavaScript and testing it out in IE11 (and maybe creating an isolated test case—hey, maybe even a browser test?) before I can come up with a more accurate estimate of how long it'll take to resolve.
